mpdev/internal/resources/deployment_manager.go

```diff
@@ -113,7 +113,7 @@ func (dm *DeploymentManagerAutogenTemplate) Test(registry Registry, dryRun bool)
 }
 
 func (dm *DeploymentManagerAutogenTemplate) runAutogen(registry Registry, inputDir string) error {
-	autogenImg := "gcr.io/cloud-marketplace-tools/dm/autogen"
+	autogenImg := "$ARTIFACT_REGISTRY/cloud-marketplace-tools/dm/autogen"
```

Comment:
how is the user going to replace this variable? Is there another entry point for them to set this image location?